APE9.rst

```diff
@@ -0,0 +1,111 @@
+A General Timeseries Class for Astropy
+-------------------------------------
+
+author: Stuart Mumford
+
+date-created: 2016 March 24
+
+date-last-revised: 2016 March 24
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+The goal of a timeseries object in astropy is to provide a core set of
+functionality that can be shared between astropy, affiliated packages and the
+wider community.
+
+The scope of this timeseries object is designed to remain focused on the core
```

Comment:
Reword suggestion, `the scope of this time series object is to provide the capability to support a unit and meta-data aware time series that supports basic analysis tasks relevant to the community. Such an object should also be able to easily export itself into other formats such as pandas series`